doc/user/data/sql_funcs.yml

```diff
@@ -30,6 +30,12 @@
       Returns `numeric` if `x` is `int`, `double` if `x` is `real`, else returns
       same type as `x`.
 
+  - signature: 'bool_and(x: T) -> T'
+    description: True if all values of `x` are true.
```

Comment:
Unless there is a pre-existing norm, I have a preference for more explicit language here, spelling out when the function is false and when it is null. Like, I think on empty inputs the output is `NULL` which is surprising for people who do math. The description could instead be "True if all values and at least one value of `x` are true" which reads terribly but is more accurate.

Reply:
Reworded to "`NULL` if all values of `x` are `NULL`, otherwise true if all values of `x` are true, otherwise false.", which I think is as precise but reads a bit better. But please confirm that you agree this is as precise!